<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ngl Qt Demo</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
*
{
  background: #262626;
  color: #FFFFFF;
  border-color: #787878;
}
  </string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="s_mainWindowSplitter">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QSplitter" name="s_menuSplitter">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuObject">
    <property name="title">
     <string>Object</string>
    </property>
    <widget class="QMenu" name="menuAdd">
     <property name="title">
      <string>Add Primitive</string>
     </property>
     <addaction name="actionPlane"/>
     <addaction name="actionSphere"/>
     <addaction name="actionCube"/>
     <addaction name="actionCylinder"/>
     <addaction name="actionTorus"/>
     <addaction name="actionTeapot"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuAdd_Light">
     <property name="title">
      <string>Add Light</string>
     </property>
     <addaction name="actionPointLight"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionLoad"/>
    <addaction name="menuAdd"/>
    <addaction name="menuAdd_Light"/>
   </widget>
   <widget class="QMenu" name="menuScene">
    <property name="title">
     <string>Scene</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClear_Scene"/>
    <addaction name="separator"/>
    <addaction name="actionCornell"/>
    <addaction name="actionSad_Sponza"/>
    <addaction name="actionHallway"/>
    <addaction name="actionPBR"/>
    <addaction name="actionLights"/>
   </widget>
   <addaction name="menuObject"/>
   <addaction name="menuScene"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load...</string>
   </property>
  </action>
  <action name="actionSphere">
   <property name="text">
    <string>Sphere</string>
   </property>
  </action>
  <action name="actionTeapot">
   <property name="text">
    <string>Teapot</string>
   </property>
  </action>
  <action name="actionCube">
   <property name="text">
    <string>Cube</string>
   </property>
  </action>
  <action name="actionPlane">
   <property name="text">
    <string>Plane</string>
   </property>
  </action>
  <action name="actionCylinder">
   <property name="text">
    <string>Cylinder</string>
   </property>
  </action>
  <action name="actionTorus">
   <property name="text">
    <string>Torus</string>
   </property>
  </action>
  <action name="actionPointLight">
   <property name="text">
    <string>Point Light</string>
   </property>
  </action>
  <action name="actionCornell">
   <property name="text">
    <string>Cornell</string>
   </property>
  </action>
  <action name="actionSad_Sponza">
   <property name="text">
    <string>Sad Sponza</string>
   </property>
  </action>
  <action name="actionPBR">
   <property name="text">
    <string>PBR Preview</string>
   </property>
  </action>
  <action name="actionLights">
   <property name="text">
    <string>Light Stress Test</string>
   </property>
  </action>
  <action name="actionClear_Scene">
   <property name="text">
    <string>Clear Scene</string>
   </property>
  </action>
  <action name="actionHallway">
   <property name="text">
    <string>Hallway</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
